nebula-exchange/src/main/scala/com/vesoft/nebula/exchange/Exchange.scala

```diff
@@ -146,13 +149,12 @@ object Exchange {
           if (tagConfig.dataSinkConfigEntry.category == SinkCategory.CLIENT) {
             LOG.info(s"Client-Import: batchSuccess.${tagConfig.name}: ${batchSuccess.value}")
             LOG.info(s"Client-Import: batchFailure.${tagConfig.name}: ${batchFailure.value}")
+            failures += batchFailure.value
           } else {
             LOG.info(s"SST-Import: failure.${tagConfig.name}: ${batchFailure.value}")
           }
         }
       }
-    } else {
```

Comment:
This warning message should be kept ⚠️

Reply:
If user doesn't config tags, why print this warning message?  Maybe it misleads users into thinking that there is no tag created in nebula.